packages/dashboard-core-plugins/src/ChartPluginConfig.ts

```diff
@@ -1,10 +1,15 @@
-import { PluginType, DashboardPlugin } from '@deephaven/plugin';
+import { PluginType, type WidgetPlugin } from '@deephaven/plugin';
+import { vsGraph } from '@deephaven/icons';
 import ChartPlugin from './ChartPlugin';
 
-const ChartPluginConfig: DashboardPlugin = {
-  name: 'ChartPlugin',
-  type: PluginType.DASHBOARD_PLUGIN,
+const ChartPluginConfig: WidgetPlugin = {
+  name: 'ChartPanel',
+  title: 'Chart',
+  type: PluginType.WIDGET_PLUGIN,
   component: ChartPlugin,
+  panelComponent: ChartPlugin,
+  supportedTypes: 'Figure',
```

Comment:
Hm... so we can't use the `dh.VariableType.FIGURE` here anymore... Just making a note of that.

Reply:
Yep. One downside to this, but also something that would be required for lazy loading from the manifest file for 3rd party plugins.

We could potentially provide a `createConfig` function option that gets the JS API or something, but not sure it adds much. Or do something where these are registered to use the JS API values instead, but also not sure that's really worth much